server/src/main/java/keywhiz/service/crypto/RowHmacGenerator.java

```diff
@@ -36,6 +37,18 @@ public String computeRowHmac(String table, List<Object> fields) {
     return cryptographer.computeHmacWithSecretKey(hmacContent.getBytes(UTF_8), hmacKey);
   }
 
+  /**
+   * Checks whether two HMACs are equal.
+   *
+   * Uses MessageDigest to prevent timing attacks.
+   * */
+  public boolean compareHmacs(String left, String right) {
```

Comment:
I don't know Java well, so I might be totally off base here, but can/should this method be static since it doesn't need to use any of the class members?